File: app/src/main/java/me/wsj/fengyun/utils/ContentUtil.java
package me.wsj.fengyun.utils;


import android.content.Context;
import android.content.res.Resources;
import android.os.Build;
import android.provider.Settings;

import androidx.preference.PreferenceManager;

import java.lang.reflect.Method;

import me.wsj.fengyun.MyApp;

/**
 * Created by niuchong on 2019/4/7.
 */

public class ContentUtil {

    //应用设置里的文字
//    public static String SYS_LANG = "zh";
    public static String APP_SETTING_UNIT = PreferenceManager.getDefaultSharedPreferences(MyApp.Companion.getContext()).getString("unit", "she");

    public static boolean UNIT_CHANGE = false;
    public static boolean CITY_CHANGE = false;

    public static int visibleHeight = 0;
}

File: app/src/main/java/me/wsj/fengyun/utils/ContentUtil.kt
package me.wsj.fengyun.utils

import androidx.preference.PreferenceManager
import me.wsj.fengyun.MyApp.Companion.context
import me.wsj.fengyun.MyApp

/**
 * Created by niuchong on 2019/4/7.
 */
object ContentUtil {
    //应用设置里的文字
    //    public static String SYS_LANG = "zh";
    /** 单位的值(摄氏度/华氏度)。 */
    @JvmField
    var APP_SETTING_UNIT =
        PreferenceManager.getDefaultSharedPreferences(context).getString("unit", "she")
    /** 单位(摄氏度/华氏度)。 */
    @JvmField
    var UNIT_CHANGE = false
    /** 城市是否改变。 */
    @JvmField
    var CITY_CHANGE = false
    @JvmField
    var visibleHeight = 0
}